src/test/java/com/exasol/adapter/dialects/db2/IntegrationTestConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package com.exasol.adapter.dialects.db2;
+
+import java.nio.file.Path;
+
+public class IntegrationTestConfiguration {
+    public static final String EXASOL_DOCKER_REFERENCE = "7.0.7";
```

Comment:
I wonder if we also want to test this with other versions, setting up a matrix in the travis yml file

Reply:
I would do it after we add a few more tests to the class, not in this pull request